tests/src/Functional/TypedDataFormWidget/TextareaWidgetTest.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace Drupal\Tests\typed_data\Functional\TypedDataFormWidget;
+
+use Drupal\Core\Plugin\Context\ContextDefinition;
+use Drupal\Core\TypedData\DataDefinition;
+use Drupal\Core\TypedData\ListDataDefinition;
+use Drupal\Core\TypedData\MapDataDefinition;
+use Drupal\Core\TypedData\TypedDataTrait;
+use Drupal\Tests\BrowserTestBase;
+use Drupal\Tests\typed_data\BrowserTestHelpersTrait;
+use Drupal\typed_data\Util\StateTrait;
+use Drupal\typed_data\Widget\FormWidgetManagerTrait;
+
+/**
+ * Class TextInputWidgetTest.
+ *
+ * @group typed_data
+ *
+ * @coversDefaultClass \Drupal\typed_data\Plugin\TypedDataFormWidget\TextareaWidget
+ */
+class TextareaWidgetTest extends BrowserTestBase {
+
+  use BrowserTestHelpersTrait;
+  use FormWidgetManagerTrait;
+  use StateTrait;
+  use TypedDataTrait;
+
+  /**
+   * The tested form widget.
+   *
+   * @var \Drupal\typed_data\Widget\FormWidgetInterface
+   */
+  protected $widget;
+
+  /**
+   * Modules to enable.
+   *
+   * @var array
+   */
+  public static $modules = [
+    'typed_data',
+    'typed_data_widget_test',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setUp() {
+    parent::setUp();
+    $this->widget = $this->getFormWidgetManager()->createInstance('textarea');
+  }
+
+  /**
+   * @covers ::isApplicable
+   */
+  public function testIsApplicable() {
+    $this->assertFalse($this->widget->isApplicable(DataDefinition::create('any')));
+    $this->assertFalse($this->widget->isApplicable(DataDefinition::create('binary')));
+    $this->assertFalse($this->widget->isApplicable(DataDefinition::create('boolean')));
+    $this->assertTrue($this->widget->isApplicable(DataDefinition::create('datetime_iso8601')));;
```

Comment:
I don't think this makes sense - a textarea for date strings does not fit very well. I think should be only be applicable to the 'string' data type for now.

Reply:
still the case.